src/drafts/Dialog/Dialog.tsx

```diff
@@ -382,12 +371,12 @@ const Buttons: React.FC<React.PropsWithChildren<{buttons: DialogButtonProps[]}>>
   return (
     <>
       {buttons.map((dialogButtonProps, index) => {
-        const {content, buttonType = 'default', autoFocus = false, ...buttonProps} = dialogButtonProps
+        const {content, buttonType = 'normal', autoFocus = false, ...buttonProps} = dialogButtonProps
```

Comment:
Just wanted to make sure if you are aware of this [PR](https://github.com/primer/react/pull/3084) that is recently merged to main? 

Because we are moving the source code to a different folder, I am not sure how git handles moving updates. I imagine not greatly 😅

Reply:
If you move them with `git mv`, it handles them pretty well! I saw that PR and I found the changes on these Button lines a little confusing. I went with something that made more sense to _me_, but I'm happy to go with the changes from that PR if you or other reviewers prefer them.

Reply:
Well, [that PR](https://github.com/primer/react/pull/3084) is reverted due to introducing breaking changes to dotcom :( Do you think we can re-introduce it within your PR?